operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -349,37 +349,51 @@ Use boot orchestration to power on and boot the nodes. Specify the appropriate B
-      ```bash
-      ncn-m001# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
-      ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json|jq '.boot_sets[] | select(.node_list)'
-      ```
+    
+    ```bash
+    ncn-m001# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
+    ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json|jq '.boot_sets[] | select(.node_list)'
+    ```
+    
     If this query returns empty, then skip to sub-step 3.
     If this query returns with data, then one or more boot sets within the BOS Session template reference nodes explicitly by xname. Consider adding your new nodes to this list (sub-step 1) or adding them on the command line (sub-step 2).
 
     1. Adding new nodes to your list.
-          1. Dump the current Session template.
-             ```bash
-             ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json > tmp.txt
-             ```
-          2. Edit the tmp.txt file adding the new nodes to the node_list.
-             ```bash
-             ncn-m001# vi tmp.txt
-             ```
+       
+       1. Dump the current Session template.
+          
+          ```bash
+          ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json > tmp.txt
+          ```
+       
+       2. Edit the tmp.txt file adding the new nodes to the node_list.
+          
+          ```bash
+          ncn-m001# vi tmp.txt
+          ```
+
     2. Create the Session template.
-         1. The name of the Session template is determined by the name provided to the '--name' option on the command line. Use the current value of $BOS_TEMPLATE if you want to overwrite the existing Session template. If you want to use the current value, skip this sub-step and go on to sub-step 2. Otherwise, provide a different name for BOS_TEMPLATE which will be used the '--name' option. The name specified in tmp.txt is overridden by the value provided by the '--name' option.
-  	    ```bash
-	    ncn-m001# $BOS_TEMPLATE=<New Session Template name>
-	    ```
-	 2. Create the Session template.
-            ```bash
-            ncn-m001# cray bos sessiontemplate create --file tmp.txt --name $BOS_TEMPLATE
-            ```
-         3. Verify that the Session template contains the additional nodes and the proper name.
-            ```bash
-            ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json
-            ```
-
-    3. Boot the nodes.
+       
+       1. The name of the Session template is determined by the name provided to the '--name' option on the command line. Use the current value of $BOS_TEMPLATE if you want to overwrite the existing Session template. If you want to use the current value, skip this sub-step and go on to sub-step 2. Otherwise, provide a different name for BOS_TEMPLATE which will be used the '--name' option. The name specified in tmp.txt is overridden by the value provided by the '--name' option.
```

Comment:
```suggestion
       1. The name of the Session template is determined by the name provided to the `--name` option on the command line. Use the current value of `$BOS_TEMPLATE` if you want to overwrite the existing Session template. If you want to use the current value, skip this sub-step and go on to sub-step 2. Otherwise, provide a different name for `BOS_TEMPLATE` which will be used the `--name` option. The name specified in `tmp.txt` is overridden by the value provided to the `--name` option.
```